client/src/components/Ticketing/ticketingmanager/AdminPurchase/AdminCheckout.tsx

```diff
@@ -22,62 +24,64 @@ const stripePromise = loadStripe(pk);
  *
  * @returns {ReactElement}
  */
-export default function CheckoutPage(): ReactElement {
+export default function AdminCheckout(): ReactElement {
+  const location = useLocation();
+  const eventDataFromPurchase = location.state?.eventData || [];
+  console.log('eventDataFromPurchase:', eventDataFromPurchase);
   const navigate = useNavigate();
-  const cartItems = useAppSelector(selectCartContents);
+  const cartItems = location.state?.cartItems || [];
   const discount = useAppSelector(selectDiscount);
-  const donation = useAppSelector(selectDonation);
   const handleBackButton = () => {
     navigate('/ticketing/purchaseticket');
   };
+  const dispatch = useAppDispatch();
+
   const doCheckout = async (formData: CheckoutFormInfo) => {
-    if (formData.seatingAcc === 'Other') {
-      formData.seatingAcc = formData.comments;
-    }
-    const stripe = await stripePromise;
-    if (!stripe) return;
-    const response = await fetch(
-      process.env.REACT_APP_API_2_URL + `/events/checkout`,
-      {
-        credentials: 'include',
-        method: 'POST',
-        headers: {
-          'Content-Type': 'application/json',
+    try {
+      if (formData.seatingAcc === 'Other') {
+        formData.seatingAcc = formData.comments;
+      }
+      const donation = formData.donation;
+      const stripe = await stripePromise;
+      if (!stripe) return;
+      const response = await fetch(
+        process.env.REACT_APP_API_2_URL + `/events/checkout`,
+        {
+          credentials: 'include',
+          method: 'POST',
+          headers: {
+            'Content-Type': 'application/json',
+          },
+          body: JSON.stringify({cartItems, formData, donation, discount}),
         },
-        body: JSON.stringify({cartItems, formData, donation, discount}),
-      },
-    );
-    const session = await response.json();
-    const paymentIntent = session.payment_intent;
-    const result = await stripe.redirectToCheckout({sessionId: session.id});
-    if (result.error) {
-      console.error(result.error.message);
+      );
+      if (!response.ok) {
+        throw response;
+      }
+      const session = await response.json();
+      if (session.id === 'comp') {
+        dispatch(removeAllTicketsFromCart());
+        navigate(`/success`);
+        // return <CheckoutSuccess />;
```

Comment:
Should this be removed or uncommented?

Reply:
I'll remove it for now.  I was trying to fix the back button route for the success page